packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -130,7 +131,6 @@ class SelectableTableRows extends React.Component {
 
 SelectableTableRows.propTypes = propTypes;
 SelectableTableRows.defaultProps = defaultProps;
-SelectableTableRows.Row = TableRow;
```

Comment:
probably want to keep this in here for passivity

Reply:
Yes. We will want to keep it. 

SelectableTableRows.Row gives consumers access to the TableRow component directly through the SelectableTable rows. We provide this type of consumption for several packages that use multiple components to compose the full structure. Here are a examples:

- [ButtonGroup.Button](https://github.com/cerner/terra-core/blob/master/packages/terra-button-group/src/ButtonGroup.jsx#L136)

- [Menu.Item or Menu.ItemGroup or Menu.Divider](https://github.com/cerner/terra-core/blob/master/packages/terra-menu/src/Menu.jsx#L190)

Reply:
Makes sense. All three table nightwatch tests fail on the initial table render with this export line included -- looking more into it.

Reply:
What are your test failures?

Reply:
<img width="836" alt="screen shot 2017-10-11 at 12 07 56 pm" src="https://user-images.githubusercontent.com/14063284/31455294-06cf79aa-ae7d-11e7-8603-cb34d0b4eed0.png">

Reply:
Meh, thinking something is only wrong on my local build. Pushing to see what Travis says.